File: microservice-transaction-sample/client/src/main/java/sample/client/command/RepaymentCommand.java
package sample.client.command;

import com.google.protobuf.Empty;
import io.grpc.ManagedChannel;
import java.util.concurrent.Callable;
import picocli.CommandLine.Command;
import picocli.CommandLine.Parameters;
import sample.rpc.CustomerServiceGrpc;
import sample.rpc.RepaymentRequest;

@Command(name = "Repayment", description = "Repayment")
public class RepaymentCommand implements Callable<Integer> {

  @Parameters(index = "0", paramLabel = "CUSTOMER_ID", description = "customer ID")
  private int customerId;

  @Parameters(index = "1", paramLabel = "AMOUNT", description = "amount of the money for repayment")
  private int amount;

  @Override
  public Integer call() {
    ManagedChannel channel = Utils.getCustomerServiceChannel();
    try {
      CustomerServiceGrpc.CustomerServiceBlockingStub stub =
          CustomerServiceGrpc.newBlockingStub(channel);
      Empty response =
          stub.repayment(
              RepaymentRequest.newBuilder().setCustomerId(customerId).setAmount(amount).build());
      Utils.printJsonString(response);
      return 0;
    } catch (Exception e) {
      e.printStackTrace();
      return 1;
    } finally {
      Utils.shutdownChannel(channel);
    }
  }
}

File: scalardb-dotnet-samples/scalardb-cluster-linq-sample/Commands/RepaymentCommand.cs
using System.CommandLine;

namespace ScalarDbClusterLinqSample.Commands;

public static class RepaymentCommand
{
    private const string Name = "Repayment";
    private const string Description = "Repayment";

    private const string ArgName1 = "customer_id";
    private const string ArgDescription1 = "customer ID";
    private const string ArgName2 = "amount";
    private const string ArgDescription2 = "amount of the money for repayment";

    public static Command Create()
    {
        var customerIdArg = new Argument<int>(ArgName1, ArgDescription1);
        var amountArg = new Argument<int>(ArgName2, ArgDescription2);
        var repaymentCommand = new Command(Name, Description)
        {
            customerIdArg,
            amountArg
        };

        repaymentCommand.SetHandler(async (customerId, amount) =>
        {
            using var sample = new Sample();
            await sample.Repayment(customerId, amount);
        }, customerIdArg, amountArg);

        return repaymentCommand;
    }
}
